test-api-generator/citrus-test-api-generator-core/src/test/resources/apis/BookService-generated.yaml

```diff
@@ -0,0 +1,43 @@
+---
+info:
+    contact:
+        name: "org.citrusframework.openapi.generator.SimpleWsdlToOpenApiTransformer"
+    description: "This api has been generated from the following wsdl 'BookService.wsdl'.\
+    \ It's purpose is solely to serve as input for SOAP API generation. Note that\
+    \ only operations are extracted from the WSDL. No schema information whatsoever\
+    \ is generated!"
+    title: "Generated api from wsdl"
+    version: "1.0.0"
+openapi: "3.0.1"
+paths:
+    /GetBook:
+        post:
+            description: "This operation retrieves details for a specific book identified\
+        \ by its ID."
```

Comment:
this backslash looks strange. It's because you're breaking the string, right? but it should be there only once (line 15)! or even better:

```yaml
description: |
  multiline
  description
```

was this intentional?

Reply:
This is a generated file. I just copied it here. However, the slashes result from exporting the OpenAPI to yaml. I have changed exporter options to minimize quotes which in turn removes the slashes.

Reply:
thank you very much.